src/mmw/apps/modeling/views.py

```diff
@@ -235,8 +237,17 @@ def choose_worker():
     def predicate(worker_name):
         return settings.STACK_COLOR in worker_name or 'debug' in worker_name
 
+    @retry(Exception, delay=0.5, backoff=2, tries=3)
```

Comment:
Without any changes, this is a big improvement. We should wait until the error occurs again before tweaking these.